egs2/slue-voxpopuli/asr1/README.md

```diff
@@ -25,3 +25,12 @@
 |dataset|Snt|Macro F1(%)|Micro F1 (%)|Macro Label F1(%)|Micro Label F1 (%)| WER|
 |---|---|---|---|---|---|---|
 |decode_asr_asr_model_valid.acc.ave/devel|1742|61.0|74.5|81.6|88.0|9.3|
+
+## Using XLS-R pretrained speech Encoder and mBART-50 Large pretrained text Encoder-Decoder
+
+- ASR config: [conf/tuning/train_asr_branchformer_xlsr_mbart.yaml](conf/tuning/train_asr_branchformer_xlsr_mbart.yaml)
```

Comment:
Could you try to also upload this model to hugging face? It would be useful for future experimentation on this.

Reply:
Some time ago I tried to upload SLURP model and the file size was too large. I can try it again once the WavLM experiment will finish. If this does not work, I could share it somehow else (Zenodo?).

Reply:
Yeah zenodo will work too if hugging face gives some issues.